Sources/MapboxNavigation/NavigationCameraStateTransition.swift

```diff
@@ -228,8 +230,10 @@ public class NavigationCameraStateTransition: CameraStateTransition {
             animatorPadding
         ]
         
-        animators.forEach {
-            $0?.stopAnimation()
+        animators.compactMap({ $0 }).forEach {
+            if $0.isRunning {
+                $0.stopAnimation()
```

Comment:
@macdrevx suggests `CameraManager.cancelAnimations()`, which might be more elegant.

Reply:
There is a chance that `CameraManager.cancelAnimations()` will stop other animators, which are not related to navigation camera. So for now I'd rather stop only animations, which were created in scope of `NavigationCameraStateTransition.swift`.